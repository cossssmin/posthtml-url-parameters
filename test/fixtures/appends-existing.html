<a href="https://example.com?s=test">Test</a>
