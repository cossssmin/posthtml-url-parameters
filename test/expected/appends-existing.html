<a href="https://example.com?baz=qux&foo=bar&s=test">Test</a>
